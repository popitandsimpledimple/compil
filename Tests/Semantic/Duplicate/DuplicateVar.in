var a, a: integer;
begin end.