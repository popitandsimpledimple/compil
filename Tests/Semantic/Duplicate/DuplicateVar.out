(1,8) Duplicate identifier 'a'